Longest_Increasing_Subsequence/LIS.php

```diff
@@ -0,0 +1,44 @@
+<?php
+// PHP Implementation of LIS using Dynamic Programming
+
+// fumction to return the length of LIS
+function lisLen($arr, $n){
+    // $dp[$i] represents length of LIS ending at index i
```

Comment:
remove this commend

Reply:
Made the second required change in the PR. Left the first one as discussed with you @asha15 
Kindly review it.